django/apps/config.py

```diff
@@ -201,3 +201,4 @@ def ready(self):
         """
         Override this method in subclasses to run code when Django starts.
         """
+        pass
```

Comment:
Is there a reason to add a `pass` statement here?

The docstring should be enough, you can take a look at [Why does python allow an empty function (with doc-string) body without a “pass” statement? (stackoverflow)](https://stackoverflow.com/a/17735171).

Reply:
But why use some obscure feature of python.  why not use pass statement? 
I would argue having a pass statement evolved into a pattern for empty function/methods.  So why not add it?

Reply:
I think Django has generally included `pass` (perhaps a full audit could be done). I'm not sure that omitting is considered "an obscure feature" and I don't have a particular preference one way or another.

Reply:
My view is that Python design specifically allows to omit 'pass' when docstring is present, in other words it's not some corner case or side effect, it's a style and design decision that a docstring is considered as "body".

It's not a strong preference, but I'd prefer if django did not enforce adding 'pass'.